src/redux/features/accessions/accessionsThunks.ts

```diff
@@ -0,0 +1,66 @@
+import { Dispatch } from 'redux';
+import { RootState } from 'src/redux/rootReducer';
+import { SearchNodePayload } from 'src/types/Search';
+import strings from 'src/strings';
+import SeedBankService from 'src/services/SeedBankService';
+import { setAccessionsAction } from './accessionsSlice';
+
+const SEARCH_FIELDS_ACCESSIONS = ['id', 'accessionNumber', 'speciesName'];
+
+export type SearchResponseAccession = {
+  id: string;
+  accessionNumber: string;
+  speciesName: string;
+};
+
+export const requestAccessions = (organizationId: number, speciesId?: number) => {
```

Comment:
I went with a traditional thunk instead of the `createAsyncThunk` because I am trying to share the data across views using a predictable key (org ID and species ID)